src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php

```diff
@@ -37,11 +34,6 @@ class ExistsFilter extends AbstractFilter
 {
     const QUERY_PARAMETER_KEY = 'exists';
 
-    public function __construct(ManagerRegistry $managerRegistry, RequestStack $requestStack, LoggerInterface $logger = null, array $properties = null)
```

Comment:
I've seen this in other filters I think. Do this need to be part of this commit?